pixart.js

```diff
@@ -1 +1,36 @@
+var brush = $(".brush");
+var button = document.getElementById("set-color");
+var input = document.getElementById("color-field");
+function changeBrushColor (inputColor) {
+  brush.css("background", inputColor);
+}
 
+
+//I got very stuck on the part where I needed to enter the input.value, so I refered to the
+//solutions, but I'm not sure what the function(event) part of this does.
+
+button.addEventListener("click", function(event){
+  event.preventDefault();
+  changeBrushColor(input.value);
```

Comment:
You can use `.val()` in place of `.value` or `.value()`. 
- just `.value` will return the `value` property `input`.
- using `.value()` is a vanilla JS *getter function* that retrieves the `value` html attribute of a DOM element
- `.val()` is the jQuery method that does the same thing as above